.github/workflows/docker.yml

```diff
@@ -56,7 +57,7 @@ jobs:
     - name: Build and push
       uses: docker/build-push-action@a66e35b9cbcf4ad0ea91ffcaf7bbad63ad9e0229
       with:
-        push: true
+        push: false
```

Comment:
Is this supposed to have changed? Or is it just for testing?

If it's of any use on `hathor-core` there is an example of building the image, testing if the image works, and only then pushing the image.